docs/Development Concepts/Protocols/2.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Protocol Identifiers
+---
+
+Currently the main protocol identifiers used for Waku are:
+
+- using `WAKU2-RELAY`/`WAKU2-FILTER` nodes for live message publish/subscribe on a content topic
+  - `WAKU2-FILTER` is a lightweight version of `WAKU2-FILTER` specifically designed for bandwidth restricted devices.
+  - For more information, you can read the specifications for [`WAKU2-FILTER`](https://rfc.vac.dev/spec/12/) and [`WAKU2-RELAY`](https://rfc.vac.dev/spec/11/)
+- using `waku-store` nodes to save/retrieve persisting messages
+
+### Waku Relay: `/vac/waku/relay/2.0.0`
+
+`WAKU2-RELAY` specifies a Publish/Subscribe approach to peer-to-peer messaging with a strong focus on privacy, censorship-resistance, security and scalability.
```

Comment:
```suggestion
`WAKU2-RELAY` specifies a Publish/Subscribe approach to peer-to-peer messaging with a focus on privacy, censorship-resistance, security and scalability.
```

Relay is more privacy preserving than filter/lightpush/store but it does not have strong privacy guarantees.
That is why we are working on Dandelion and also possibly putting tor or nym between Waku nodes.